libs/core/src/lib/list/list-item.directive.ts

```diff
@@ -27,4 +28,25 @@ export class ListItemDirective {
     @HostBinding('class.is-selected')
     selected: boolean = false;
 
+    @ContentChild(CheckboxComponent, {static: false})
+    checkboxComponent: CheckboxComponent;
+
+    /** @hidden */
+    constructor (public itemEl: ElementRef) {}
+
+    public focus(): void {
```

Comment:
Typedocs missing for this method

Reply:
These are just convenience methods, so i'll @hidden them